### DISCUSSION TITLE
Timeline

### DISCUSSION THREAD

**Discussion by tomyems** [UKHomeOffice/design-system#442] [General]
To discuss research findings and design improvements to the [timeline component](https://design.homeoffice.gov.uk/components?name=Timeline).

**Comment by jbbrwcky** [comment]
Hi, using this in a govt site, looks good. One thing we wanted to do was to have a link in the description but no matter how we place the text in, as direct text, or component, it always displays out in full text like:
`<a href="http://abc.com/">abc</a>`
Is there not a way to make the link appear as you would expect, and clickable?
